test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-# Unit test scripts.
-set(TEST_SCRIPT_FILES
```

Comment:
The change is mainly add a non interactive section which is likely not to be usable with maya hydra tests. They all require a GUI (to use Qt) or need a viewport to set HdStorm.